app/controllers/EightBallController.scala

```diff
@@ -0,0 +1,30 @@
+package controllers
+
+import javax.inject._
+
+import play.api.mvc._
+
+import models.EightBallReply
+
+@Singleton
+class EightBallController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
+
+  def randomReply = Action {
+    val rndString = EightBallReply.getRndBall.content
+    Ok(views.txt.text(rndString))
+  }
+
+  def getMessages = Action {
+    val allMsgs = (EightBallReply.getAll map ( x => x.content ) ).
+      fold("")((x:String, y:String) => x + "\n" + y)
```

Comment:
Just for ref  https://docs.scala-lang.org/style/declarations.html
"Function Values" section